packages/integrations/node/src/http-server.ts

```diff
@@ -54,6 +62,12 @@ export function createServer(
 				// File not found, forward to the SSR handler
 				handler(req, res);
 			});
+			stream.on('headers', (_res: http.ServerResponse<http.IncomingMessage>) => {
+				if(isImmutableAsset(encodedURI)) {
+					// Taken from https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#immutable
+					_res.setHeader('Cache-Control', 'public, max-age=604800, immutable')
```

Comment:
Perhaps we can also align the value as Vercel's  https://github.com/withastro/astro/blob/e3dce215a5ea06bcff1b21027e5613e6518c69d4/packages/integrations/vercel/src/static/adapter.ts#L128